origami.json

```diff
@@ -11,13 +11,14 @@
 	"demosDefaults": {
 		"sass": "demos/src/main.scss",
 		"js": "demos/src/demo.js",
-		"template": "demos/src/video.mustache"
+		"dependencies": "o-fonts@^2.1.0"
 	},
 	"demos": [
 		{
 			"name": "native",
 			"description": "Native",
-			"expanded": true
+			"expanded": true,
```

Comment:
This property is deprecated and should be removed. The registry no longer listens to this property.